README.md

```diff
@@ -11,25 +9,71 @@ TESTING
 ![Python package](https://github.com/nasa/harmony-py/workflows/Python%20package/badge.svg)
 
 ---
+# Using Harmony Py
+
+## Prerequisites
+
+* Python 3.7+
+
 
 ## Installing
 
-The library is available from Pypi and can be installed with pip:
+The library is available from [PyPI](#TODO) and can be installed with pip:
```

Comment:
Yes this wasn't obvious in my testing that I wasn't able to install via PyPI yet.